lib/active_record/connection_adapters/cockroachdb_adapter.rb

```diff
@@ -83,8 +83,7 @@ def supports_materialized_views?
       end
 
       def supports_partial_index?
-        # See cockroachdb/cockroach#9683
-        false
+        cockroachdb_version >= 202
```

Comment:
I think you can just do `@crdb_version >= 202` here and forego the `cockroachdb_version` method entirely.

Reply:
that seems right. i just did it this way since i saw that's what the postgres_adapter does (https://github.com/rails/rails/blob/2a3f8a29e5d7d17a0448fa47fd756c8d11b175b1/activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb#L400)

but don't think there's a particular reason to do so